pkg/agent/cniserver/types/types.go

```diff
@@ -41,12 +43,23 @@ type Range struct {
 
 type RangeSet []Range
 
+type Address struct {
+	Address string `json:"address"`
+	Gateway net.IP `json:"gateway,omitempty"`
+	IPNet   net.IPNet
+	// IP version. Either "4" or "6".
+	Version string
```

Comment:
I think this is an internal type. Could we use enum instead of string?

Reply:
Yes, you can say it is an internal type, but we never read or check the version value. Its only use is to generate the IPAM result which uses string (see: https://github.com/antrea-io/antrea/pull/3633/files#diff-b0835971aa431617cb107905707afd58dc43d6a495d3b17bc7c8965bfbfc8578R270). So, I do not think changing to an enum helps anything.